packages/atlas/vite.config.ts

```diff
@@ -52,6 +53,15 @@ export default defineConfig({
       },
       overlay: false,
     }),
+    {
+      ...inject({
+        include: ['node_modules/**/*.js*'],
+        modules: {
+          Buffer: ['buffer', 'Buffer'],
+        },
+      }),
+      enforce: 'post',
+    },
```

Comment:
Why is this needed? I was pretty sure the `Buffer` polyfill worked properly before. Maybe with the ESM `@joystream/js` version it's not needed anymore?

Reply:
I'm still getting `Buffer is not defined` when I run `claimReward` unfortunately. I think I need this, unfortunately